.circleci/config.yml.in

```diff
@@ -287,13 +290,13 @@ jobs:
 
 
 workflows:
-  # build:
-  #   jobs:
-  #     - circleci_consistency
-  #     {{ build_workflows() }}
-  #     - binary_win_conda:
-  #         name: torchtext_win_py3.6
-  #         python_version: "3.6"
+  build:
+    jobs:
+      - circleci_consistency
+      {{ build_workflows() }}
+      #  - binary_win_conda:
```

Comment:
Is this to be fixed in a follow-up PR?

Reply:
Not my intent.
I am just keeping what was there and making `build_workflows` work.

Reply:
Should we test the torchtext package with cpp extension on a Windows machine? Although we don't explicitly support Windows, the current package does work on Windows.

Reply:
Ideally, we should. `torchaudio` does not have this but needs this soon.
We can look into how `torchvision` does it or ask help from someone with Windows experience.

Reply:
@peterjc123

Reply:
I'll add unit tests first. Binary builds on Windows will come later.